csv_file_to_dataframe.py

```diff
@@ -0,0 +1,20 @@
+import boto3
+import pandas as pd
+
+def dataframe(bucket, folder):

```

Comment:
"dataframe" is a bad name for a function, as it mirrors existing Pandas functions.  Try something like 'csv_to_dataframe'.